README.md

```diff
@@ -36,6 +36,30 @@ include ::profile_lustre::lnet_router
 
 ### Recent changes
 
+:warning: WARNING: v3.2.0 of this module introduces a change to the "trigger" file (`/etc/lnet.trigger`
+by default). When Puppet (re)configures LNet it will now not only create this file but also put the
+system boot time in the file. On subsequent Puppet runs, if the file is missing OR if the system
+boot time recorded in the file is different than the actual system boot time, then Puppet will
+necessarily attempt to unmount Lustre and unload Lustre/LNet kernel modules, and then (re)configure
+LNet.
+
+(This change in the trigger file is necessary to support stateful systems.)
+
+When introducing this updated module into a cluster with booted production systems admins may want to
+"preseed" each node's system boot time into their trigger file to prevent Puppet from (unmounting Lustre
+and unloading Lustre/LNet kernel modules and) reconfiguring LNet. This can be done by running these commands
```

Comment:
Nice. I hadn't considered this change possibly causing issues for stateless nodes.